src/feature.js

```diff
@@ -88,10 +96,16 @@ class Feature {
     browser.pageMonitor.onPageDOMContentLoaded.addListener(async (tabId, data) => {
       const tabInfo = TabRecords.getOrInsertTabInfo(tabId);
       const location = /[^?]*/.exec(data.completeLocation)[0];
+
       data.completeLocation = null;
+      tabInfo.currentDomain = location;
```

Comment:
Just to clarify, is this a domain/hostname (`www.mozilla.org`), an origin (`https://www.mozilla.org`), or a URL (`https://www.mozilla.org/path/to/the/thing`)?

Looking at https://github.com/mozilla/CookieRestrictionsBreakageStudy/blob/5c975d1cf1f3db91f7ba3d236894fa46ce76e94e/src/privileged/pageMonitor/framescript.js#L29 it seems to me like it's the URL.

In this case it might be a good idea to call this currentURL instead?

Saving the domain/hostname only would not really be compatible with deleting items from permission manager, which is why I was confused for a second until I realized this is probably the URL :)

Reply:
oh yeah, that was a recent change since I originally was just saving the domain, but I couldn't construct a URL from that for adding/removing the exception. I'll change the name